Makefile

```diff
@@ -83,6 +84,13 @@ format-xml:
 	XMLLINT_INDENT="  " && ${XML_FORMAT_EXEC} --format $${f} --output $${f} || exit 1; \
 	done
 
+format-py:
+# Format all the python scripts under this project, excluding submodules
+	for f in `find openfpga_flow/scripts -iname *.py`; \
+	do \
+	${PYTHON_FORMAT_EXEC} $${f} --line-length 100 || exit 1; \
```

Comment:
I think default is 88 right
https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length

Reply:
Correct. I will the style from VTR, which is 100.

https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/367c45c0b1084e41e7e8ce1f231ca084da27b588/cmake/modules/AutoPyFormat.cmake#L5

Reply:
ok